aglio-olio.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html> 
+<html>
+    <head>
+        <title>Aglio Olio</title>
+        <link rel="stylesheet" href="individual-styles.css">
+    </head>
+    <body>
+        <div class="main-photo">
+                      <img src="aglio-olio.jpg" alt="aglio olio" width="410">
```

Comment:
I think you need your auto formatter to be setup correctly. Doesn't seem to work